### ISSUE TITLE
add `--format none` as an alias for `--skip-download`

### ISSUE THREAD

**Issue by milahu** [yt-dlp/yt-dlp#7397]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm requesting a feature unrelated to a specific site
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2023.06.22** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

expected:

let me use

```
yt-dlp --format none
```

instead of

```
yt-dlp --skip-download
```

`--format none` is the opposite of `--format all` (Select all formats separately)

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

**Comment by pukkandan** [comment]
Why?

**Comment by milahu** [comment]
to me, `--format none` is more obvious. i had to google to find `--skip-download`

**Comment by stt** [comment]
> With --skip-download, a format is still selected and written to info-json etc. Just that it's not downloaded.

Right, I'd like to have `--format none` that ignored anything to do with video and audio when I'm only interested in downloading subtitles or some metadata (and preferably it'd then also make it skip unnecessary API requests like m3u8 etc, but guess I can try doing `--extractor-args 'youtube:player_skip=..'` to get rid of them myself).

In general it should IMO be easier to make yt-dlp less noisy and to avoid "429 Too Many Requests" when going through sources with thousands of videos.

**Comment by dirkf** [comment]
This doesn't sound completely deluded. In yt-dl we have [this request](https://github.com/ytdl-org/youtube-dl/issues/32687).

An obvious, though solvable, problem is that `none` is a valid format_id that would have to be munged to avoid confusion. But would OP want to skip downloading manifests that might contain subtitles, etc?

**Comment by milahu** [comment]
> `none` is a valid format_id

what format would that be?

> skip downloading manifests that might contain subtitles, etc?

ideally, yt-dlp should fetch only what it needs
but i guess that fetching metadata/manifests/json/... is cheap in most cases

**Comment by dirkf** [comment]
`format_id`s are initially generated by the site extractor and then disambiguated (including the special case where no ID was returned). Although YT IDs are numeric, other sites could generate any string value.

The problem with m3u8 and mpd is that you can't avoid 429 by skipping the manifest if you also want subtitles, unless the site happens to offer subtitles separately. So the feature may not achieve what you actually want for YT. Maybe more control could be offered through extractor-args, as you suggested, and possibly through new such args.
